purchase_ux/models/purchase_order_line.py

```diff
@@ -455,3 +406,22 @@ def _onchange_quantity(self):
                     to_uom_id=self.product_uom.id)
             self.price_unit = price_unit
         return res
+
+    @api.onchange('product_qty')
+    def _onchange_product_qty(self):
+        if (
+                self.state == 'purchase' and
+                self.product_id.type in ['product', 'consu'] and
+                self.product_qty < self._origin.product_qty):
+            warning_mess = {
+                'title': _('Ordered quantity decreased!'),
+                'message': (
+                    '¡Está reduciendo la cantidad pedida! Esto no está '
+                    'permitido, le recomendamos que entregue los productos '
+                    'correspondientes y luego cancele el remanente utilzando '
+                    'el botón para tal fin. Se restableció la cantidad a la '
+                    'original'),
```

Comment:
Use
```diff
+                    'You are reducing the quantity ordered! We recommend '
+                    'use the button to cancel remaining and then set'
+                    ' the desired amount.'),
```

Reply:
whats wrong with original message? I would keep spanish and that message for this time

Reply:
@jjscarafia just for standard, to have the same message of sale_stock_ux